aioshelly/const.py

```diff
@@ -123,7 +123,6 @@
 MODEL_OUT_PLUG_S_G3 = "S3PL-20112EU"
 MODEL_PM_MINI_G3 = "S3PM-001PCEU16"
 MODEL_PLUG_S_G3 = "S3PL-00112EU"
-MODEL_X_MOD1 = "S3MX-0A"
```

Comment:
What about X devices without configuration from x.shelly.cloud? JWT block is not available for them.

```
{"name":null,"id":"shellyxmod1-xxxxxxxx","mac":"xxxxxxxx","slot":0,"model":"S3MX-0A","gen":3,"fw_id":"20241011-114452/1.4.4-g6d2a586","ver":"1.4.4","app":"XMOD1","auth_en":false,"auth_domain":null,"jti":"000000000000"}
```

Reply:
I though that all X devices need a  X.shelly.cloud config, but seems not.
Let me ask Shelly for a clarification.

Thx for spotting.